docs/docs/workflows/create.md

```diff
@@ -36,217 +36,19 @@ constellation create
-yq eval ".infrastructure.azure.userAssignedIdentity =\"$(terraform output user_assigned_identity_client_id | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.azure.attestationURL =\"$(terraform output attestationURL | jq -r)\"" --inplace constellation-state.yaml
-```
-
-</tabItem>
-<tabItem value="gcp" label="GCP">
-
-```bash
-yq eval '.version ="v1"' --inplace constellation-state.yaml
-yq eval ".infrastructure.initSecret =\"$(terraform output initSecret | jq -r | tr -d '\n' | hexdump -ve '/1 "%02x"' && echo '')\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.clusterEndpoint =\"$(terraform output out_of_cluster_endpoint | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.inClusterEndpoint =\"$(terraform output in_cluster_endpoint | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.ipCidrNode =\"$(terraform output ip_cidr_nodes | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.uid =\"$(terraform output uid | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.name =\"$(terraform output name | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.apiServerCertSANs =$(terraform output -json api_server_cert_sans)" --inplace constellation-state.yaml
-yq eval ".infrastructure.gcp.projectID =\"$(terraform output project | jq -r)\"" --inplace constellation-state.yaml
-yq eval ".infrastructure.gcp.ipCidrPod =\"$(terraform output ip_cidr_pods | jq -r)\"" --inplace constellation-state.yaml
-```
-
-</tabItem>
-</tabs>
-</tabItem>
-<tabItem value="self-managed" label="Self-managed">
-
-Self-managed infrastructure allows for managing the cloud resources necessary for a Constellation cluster separate from the Constellation CLI.
-This provides flexibility in DevOps and can meet potential regulatory requirements.
-
-To self-manage the infrastructure of your cluster, download the Terraform files for the selected CSP from the [Constellation GitHub repository](https://github.com/edgelesssys/constellation/tree/main/terraform/infrastructure).
-They contain a minimum configuration for the resources necessary to run a Constellation cluster on the corresponding CSP. From this base, you can now add, edit, or substitute resources per your own requirements with the infrastructure
+If using a tool other than Terraform, you can still refer to the Terraform files for the selected CSP from the [Constellation GitHub repository](https://github.com/edgelesssys/constellation/tree/main/terraform/infrastructure) for a minimum Constellation cluster configuration. From this base, you can now add, edit, or substitute resources per your own requirements with the infrastructure
 management tooling of your choice. You need to keep the essential functionality of the base configuration in order for your cluster to function correctly.
```

Comment:
The rest of this section reads handwavy. Can we give more detailed instructions on how to get to the state file either here or in the terraform module page? Or am I missing something?

Reply:
@msanft since you did the self managed docs, what do you think? I agree that it's handwavy.
I'd add at least 
`config generate` - to generate the config and state file
And describe that the `Infrastructure` object of the state file, needs to be filled out by the user.

Reply:
But I also think that we don't need to be too explicit here, since we don't really know of any customer not using Terraform at the moment right?

Reply:
what do you mean? The necessity to fill out the fields is mentioned here:
https://github.com/edgelesssys/constellation/blob/48e166419518bdf68488a25555140d2438c0d958/docs/docs/workflows/create.md#L68

Reply:
Ah. This is what I meant. I must have missed this in my first review

Reply:
This should be alright then.